src/libraries/System.Private.CoreLib/src/System/Runtime/CompilerServices/StackAllocatedBox.cs

```diff
@@ -9,7 +9,7 @@ namespace System.Runtime.CompilerServices
     internal unsafe struct StackAllocatedBox<T>
     {
         // These fields are only accessed from jitted code
-        private MethodTable* _pMethodTable;
+        private IntPtr _pMethodTable;
```

Comment:
What's the problem with MethodTable*?

Reply:
> What's the problem with MethodTable*?

Afair, they're in different namespaces in CoreCLR and NAOT, is it important to keep it MethodTable* ?

Reply:
Ah ok, that's fine then.